Locales/enUS.lua

```diff
@@ -6,6 +6,7 @@ L["Discord"] = "Discord"
 L['PVE Voice Alert'] = "PVE Voice Alert"
 L["Profiles"] = "Profiles"
 L["My own abilities"] = "My own abilities"
+L["Buffs on player only"] = "Buffs on player only"
```

Comment:
"Buffs on player only" might be to generic. We should clearly state, if you toggle this button you will only hear externals cast on you.

Reply:
Updated name